Controls/Mailman.c

```diff
@@ -0,0 +1,25 @@
+#include "os.h"
+
+/**
+ * The mailman will put mail in the mailbox.
+ */
+
+
+/**
+ * @brief Puts a letter in the mailbox (declared in Tasks.h). The letter must be unique every time
+ * this function is called (suggestion: include a random number!). Print "Finished!" at the end.
+ */
+void depositLetter(void) {
+	OS_ERR err; // Make sure to check for errors and print the error code if not OS_ERR_NONE
+}
+
+/**
+ * @brief Calls depositLetter. Once the letter is deposited, signal the semaphore (effectively putting up the flag).
+ * Delay briefly (2-3 seconds) using OSTimeDlyHMSM(). Loop.
```

Comment:
You have a race condition if the recipient doesn't read the mailbox in time and you start writing to the mailbox again